imports/plugins/core/orders/client/templates/workflow/shippingTracking.js

```diff
@@ -109,6 +109,14 @@ Template.coreOrderShippingTracking.helpers({
     return completedItems;
   },
 
+  orderCancelled() {
```

Comment:
Let's improve this to a simpler method. by returning the boolean only
 e.g `const o = (check) => {return check !== 'ehehe'}`